packs/data/feats.db/crossbow-crack-shot.json

```diff
@@ -19,7 +19,38 @@
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "damage-roll",
+                "key": "RollOption",
+                "option": "crossbow-crack-shot",
+                "toggleable": true,
+                "label": "Crossbow Crack Shot",
+                "value": true
+            },
+            {
+                "key": "DamageDice",
+                "predicate": [
+                    "item:category:simple",
+                    "item:tag:crossbow",
```

Comment:
item:base:crossbow

Reply:
It should in fact be "item:tag:crossbow"